docs/getting-started/obtaining-dcr.md

```diff
@@ -21,56 +20,53 @@ The five basic ways are:
 
 ### Exchanges
 
-One way to acquire Decred is to purchase some on an exchange.
-Since the release of Decred on February 8th 2016, more than ten
-cryptocurrency exchanges have added support for Decred (last updated December 2017).
-
-* [Poloniex](https://poloniex.com/)
-* [Bittrex](https://bittrex.com/)
-* [Bleutrade](https://bleutrade.com/exchange)
-* [Tux Exchange](https://tuxexchange.com)
-* [YoBit](https://yobit.net/en/)
-* [Cryptopia](https://cryptopia.co.nz/Exchange/)
-* [Exodus](https://exodus.io)
-* [Evercoin](https://evercoin.com/)
-* [ShapeShift](https://shapeshift.io)
-* [Changelly](https://changelly.com/)
-* [Flyp](https://flyp.me)
-* [Godex](https://godex.io)
-* [Fex](https://fex.hk)
-* [Upbit](https://upbit.com)
-* [LiteBit](https://www.litebit.eu/en/buy/decred)
-* [Coinspot](https://www.coinspot.com.au/buy/dcr)
+One way to acquire Decred is to purchase some on an exchange with Bitcoin.
+Since the release of Decred on February 8th 2016, more than ten cryptocurrency exchanges have added support for Decred (last updated February 2018).
+
+Note: If you need help obtaining Bitcoin to get started, the process is well documented in the [Bitcoin wiki](https://en.bitcoin.it/wiki/Buying_Bitcoins_%28the_newbie_version%29).
+
+* [Bittrex](https://bittrex.com/Market/Index?MarketName=BTC-DCR)
+* [Poloniex](https://poloniex.com/exchange#btc_dcr)
+* [Upbit](https://www.upbit.com/exchange?code=CRIX.UPBIT.BTC-DCR)
+* [Fex](https://fex.hk/Home/Exchange?type=DCR/BTC&buytype=BTC)
 * [Zhaobi](https://www.zhaobi.com/#/usercenter/dcrUSDTtrade)
+* [Cryptopia](https://www.cryptopia.co.nz/Exchange/?market=DCR_BTC)
+* [Bleutrade](https://bleutrade.com/exchange/DCR/BTC)
+* [Tux Exchange](https://tuxexchange.com/trade?coin=DCR&market=BTC)
 * [Bisq](https://markets.bisq.network/?market=dcr_btc)
 
-If you need help obtaining Bitcoin to get started, the process is well documented in the [Bitcoin wiki](https://en.bitcoin.it/wiki/Buying_Bitcoins_%28the_newbie_version%29).
+
+### Instant Exchanges
+
+Instant exchanges allow users to convert from one cryptocurrency to another without the need for registration. Do note that higher fees usually apply, i.e., the price will include a fee for the convenience. You should compare prices among all the exchanges below.
+
+* [ShapeShift](https://shapeshift.io/)
+* [Flyp](https://flyp.me/)
+* [Godex](https://godex.io/)
+* [CoinSwitch](https://www.coinswitch.co/)
+* [Evercoin](https://evercoin.com/)
+* [BitPlace](https://www.bitplace.exchange/)
```

Comment:
We should probably remove this right?

Reply:
BitPlace is dead, I think the others are still fine unless someone reported an issue.

Reply:
Right just bitplace to remove